tests/unit/TodoList1.spec.ts

```diff
@@ -1091,27 +1100,27 @@ describe('TodoList - UI - BCOMPs', () => {
 
   xit('disables \'File Annual Report\' and verification checkbox when COA is pending', async () => {
     // init store
-    store.state.tasks = [
+    rootStore.tasks = [
       {
         'task': {
           'todo': {
             header: {
-              'name': 'annualReport',
+              'name': FilingTypes.ANNUAL_REPORT,
               'ARFilingYear': 2019,
-              'status': 'NEW'
+              'status': FilingStatus.NEW
             },
             business: {
               'nextAnnualReport': '2019-09-17T00:00:00+00:00'
             }
-          }
+          } as any
         },
         'enabled': true,
         'order': 1
       }
     ]
-    store.state.isCoaPending = true // normally set by FilingHistoryList
+    // store.state.isCoaPending = true // normally set by FilingHistoryList
```

Comment:
What's this?

Reply:
I searched the entire store and I couldn't find anything called `isCoaPending`. I wanted to remove it but thought let me comment it first. Any idea what this is or why was it added Sev?

Reply:
Oh I see -- it's not a store state variables; it's a local getter in the component.

![image](https://github.com/bcgov/business-filings-ui/assets/10002889/6e63b984-bd0e-440d-a628-3881510a2f6e)

If the test fails then you'll need to overwrite the computed value or twiddle the store so there is a pending COA filing.

Reply:
Sev, can I fix this test in the other ticket (https://github.com/bcgov/entity/issues/16392)? This is the only test that's left to fix. I'm trying to set the store so that this line returns `true` but I can't seem to get it to work: https://github.com/JazzarKarim/business-filings-ui/blob/96ead3bd0a620130d82795501935ab5c1d20d359/src/stores/filingHistoryListStore.ts#L61

I can take a deeper look at it while working on the other ticket.

Reply:
Sure, add this one as a todo checkbox in the other ticket, and then you can resolve this conversation.